.travis.yml

```diff
@@ -23,6 +23,19 @@ cache:
   directories:
   - $HOME/.yarn-cache
 
+before_install:
```

Comment:
Yes? I branched this off the last PR so maybe that is why but I'm confused because I merged this with master and it still shows as a diff?